controllers/api/api_status_controller.py

```diff
@@ -42,10 +42,13 @@ def _render(self):
         return json.dumps(status_dict, ensure_ascii=True)
 
     @classmethod
-    def clear_cache_if_needed(cls, old_content, new_content):
+    def clear_cache_if_needed(cls, old_content, new_content, users_not_to_notify=[]):
```

Comment:
How are you anticipating users_not_to_notify to work? It seems like typically we'd want the opposite? Seems weird to specify a large list of users not to notify.